modules/core/src/v2/coins/bch.ts

```diff
@@ -189,27 +190,13 @@ export class Bch extends AbstractUtxoCoin {
     return common.Environments[this.bitgo.env].bchExplorerBaseUrl + url;
   }
 
-  getAddressInfoFromExplorer(addressBase58) {
-    return co<AddressInfo>(function *getAddressInfoFromExplorer() {
-      const addrInfo = yield request.get(this.recoveryBlockchainExplorerUrl(`/addr/${addressBase58}`)).result();
```

Comment:
Can we remove `recoveryBlockchainExplorerUrl()` on lines 189-191 now?

Reply:
ah thanks, nice catch!